From 5231cef75ab62581ecaa67f604c946324ec5aeb7 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 30 Nov 2012 12:41:34 +0800
Subject: [PATCH 108/304] ENGR00230588 update build id to be BETA-RC2

Rename release 15.0.0-beta to be JB4.1.2_1.0.0-beta.
No need to include the JB version into BUILD_ID, since there is
ro.build.version.release to cover the JB version

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk     | 2 +-
 arm2_6sl/build_id.mk     | 2 +-
 imx5x/build_id.mk        | 2 +-
 sabreauto_6q/build_id.mk | 2 +-
 sabresd_6dq/build_id.mk  | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index 2bbf9fb..01184e4 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=15-BETA-RC1
+export BUILD_ID=BETA-RC2
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index 2bbf9fb..01184e4 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=15-BETA-RC1
+export BUILD_ID=BETA-RC2
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index 2bbf9fb..01184e4 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=15-BETA-RC1
+export BUILD_ID=BETA-RC2
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index 2bbf9fb..01184e4 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=15-BETA-RC1
+export BUILD_ID=BETA-RC2
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index 2bbf9fb..01184e4 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=15-BETA-RC1
+export BUILD_ID=BETA-RC2
-- 
1.8.0

